python/awwvision/worker/src/requirements.txt

```diff
@@ -1,5 +1,5 @@
 gcloud==0.8.0
-google-api-python-client==1.4.2
+google-api-python-client
```

Comment:
Could you pin to the latest working version?

I know it's a bit silly to have to keep updating this, but it's better to have something that's in a known-working state but old than it is to be on the latest, but maybe broken.